lnrpc/autopilotrpc/autopilot.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package autopilotrpc;
+
+// Autopilot is a service that can be used to get information about the current
+// state of the daemon's autopilot agent, and also supply it with information
+// that can be used when deciding where to open channels.
+service Autopilot {
+    /**
+    GetStatus returns the status of the daemon's autopilot agent.
+    */
+    rpc GetStatus(GetStatusRequest) returns (GetStatusResponse);
+
+    /**
+    Enable is used to enable or disable the autopilot agent.
+    */
+    rpc Enable(EnableRequest) returns (EnableResponse);
```

Comment:
Possible can name it something like `ModifyStatus`? As it's a bit awkward that atm you need to call `Enable` with a false bool rather than something like `Disable`.

Reply:
What do you think about `SetEnabled` or `SetActive`?

Reply:
Those could work as well, personally I lean towards `ModifyStatus` as in the future we can do things like modify the status of things like it's "operating type", so: end user, routing node, etc. Of course, this can also be a follow up later.

Reply:
Renamed RPCs to `Status` and `ModifyStatus`.